src/superset/Dockerfile

```diff
@@ -21,12 +21,12 @@ RUN pip install --upgrade pip && \
 COPY --chown=superset superset_config.py /app/superset_config.py
```

Comment:
Hi @NicholasTurner23 , thanks for this.
This file should ideally not be changed in this PR for analytics microservice. Please crosscheck your Git.